src/Service/Helper.php

```diff
@@ -324,10 +324,33 @@ public function saveEMSLog($msg, $context) {
         $ems_logger->error($msg, $context);
         }
 
+    /**
+     *  Save the IBAN information for Bank-Transfer payment metohd
+     * @param $orderTransactionId
+     * @param $payment_method_details
+     * @param $orderRepository
+     * @param $context
+     * @return mixed
+     */
+
+    public function saveIbanInfo($orderTransactionId,$payment_method_details,$orderRepository,$context){
```

Comment:
How about combining functions saveIbanInfo and saveGingerOrderId into one?

Reply:
Was done